extensions/LIZARD_OFFICIAL/aes.js

```diff
@@ -0,0 +1,59 @@
+
+  class AESE {
+    getInfo () {
+      return {
+        id: 'AES',
```

Comment:
The extension ID should contain someway to identify it's made by you. Something like "aeslizard" would work.